CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# v1.0.6
+*  Update deployment artifacts.
+   Humble Chirammal hchiramm@redhat.com
+
+# v1.0.0
```

Comment:
This is just my opinion, but let's not jump to v1 right away. Let's spend some time at v0.y.z before ending up at v1.